tests/test_fairstep.py

```diff
@@ -88,3 +90,20 @@ def test_validation():
     step = FairStep(uri='http://www.example.org/step')
     with pytest.raises(AssertionError):
         step.validate()
+
+
+@pytest.mark.flaky(max_runs=10)
+@pytest.mark.skipif(nanopub_server_unavailable(), reason=SERVER_UNAVAILABLE)
+@patch('fairworkflows.nanopub_wrapper.publish')
+def test_modification_and_republishing(nanopub_wrapper_publish_mock):
+
+    preheat_oven = FairStep(uri='http://purl.org/np/RACLlhNijmCk4AX_2PuoBPHKfY1T6jieGaUPVFv-fWCAg#step', from_nanopub=True)
```

Comment:
The only reason we need the nanopub server here is because we want to build a FairStep object. Can we not choose another method so this test becomes independent from the nanopub server being online?

Reply:
I've now mocked ```Nanopub.fetch()``` to load a locally stored trig file and return a nanopub from that. This way the test no longer requires actual access to the nanopub servers.